roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -289,6 +290,11 @@
     state: present
     name: "logging-{{ es_component }}-cluster"
     namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    content:
```

Comment:
the module doesn't seem to know about this... 
https://github.com/openshift/openshift-ansible/blob/master/roles/lib_openshift/library/oc_service.py#L1909